validation-test/stdlib/Arrays.swift.gyb

```diff
@@ -6,6 +6,12 @@ import StdlibCollectionUnittest
 
 let CopyToNativeArrayBufferTests = TestSuite("CopyToNativeArrayBufferTests")
 
+extension Array {
+  func _rawIdentifier() -> Int {
+    return unsafeBitCast(self, to: Int.self)
```

Comment:
Doesn't it result in a warning?

Reply:
No warning that I can see.

Reply:
We already do this exact same thing for the Dictionary COW tests

Reply:
Doesn't mean we should be doing it =)